server/controllers/request.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+const Request = require('../models/Request');
+const asyncHandler = require('express-async-handler');
+
+// @route GET /request
+// @desc get requests related to a logged in user or sitter
+// @access Private
+exports.getRequest = asyncHandler(async (req, res) => {
+  const userId = req.user.id;
+  const getUser = await Request.find({userId: req.user.id});
+  if(getUser){
+    res.send(200).json({ success:{ message:  "get the " + getUser }});
```

Comment:
I would suggest you use template strings `get the ${getUser}` it looks cleaner

Reply:
Thanks for the recommendation